spec/factories/consent_response.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
```

Comment:
As per the FactoryBot generator, this file should be `consent_responses.rb` (pluralised)

Reply:
yes, good spot, fixed in [7ccb50a](https://github.com/nhsuk/record-childrens-vaccinations/pull/280/commits/7ccb50a7862145297e4438e4a844ce600a044718)